core/static/js/back_to_top.js

```diff
@@ -0,0 +1,22 @@
+(() => {
+  const btn = document.querySelector(".back-to-top");  // Get the button element
+  if (!btn) return; // Exit if button is not found
+
+  const SHOW_AFTER_PX = 10; // Show button after scrolling down
+
+  // Function to toggle visibility
+  const setVisibility = () => { 
+    btn.classList.toggle("is-visible", window.scrollY > SHOW_AFTER_PX); // CSS uses .is-visible to show/hide
+  };
+
+  setVisibility(); // Initial run - in case page is loaded scrolled down
+  window.addEventListener("scroll", setVisibility, { passive: true });  // Run on scroll, passive for performance
+
+  // Scroll to top on click
```

Comment:
We don’t need the following block just for smooth scroll, we can fix that with just `scroll-behaviour` css property. But it doesn’t have to be added in this PR, it will be part of common styling fix that will be done before MVP launch.

Reply:
I can leave this here but add a comment?